config.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
   },
 
   billing: {
-    supplementaryYears: isTest ? 1 : 6,
+    supplementaryYears: isTest ? 1 : 5,
     // There are 4 processes on the environments but only 1 locally
```

Comment:
Very minor comment, is it worth assigning this number to a variable for use in the tests? Feel free to ignore.